pkg/kubelet/lifecycle/predicate.go

```diff
@@ -279,7 +279,8 @@ func (e *PredicateFailureError) GetReason() string {
 
 // generalFilter checks a group of filterings that the kubelet cares about.
 func generalFilter(pod *v1.Pod, nodeInfo *schedulerframework.NodeInfo) []PredicateFailureReason {
-	admissionResults := scheduler.AdmissionCheck(pod, nodeInfo, true)
+	admissionResults := scheduler.AdmissionCheck(pod, nodeInfo, true,
+		pod.Status.Phase == v1.PodRunning || pod.Status.PodIP != "")
```

Comment:
What really means this check of `pod.Status.PodIP != ""` ?

Reply:
Is a error check.I fix it, thanks.